### DISCUSSION TITLE
Incremental Append mode not working for Zendesk Chat source

### DISCUSSION THREAD

**Discussion by WisamulHaque** [airbytehq/airbyte#34093] [General]
Incremental | Append mode is replicating data in destination even when no new data is added in source or any update is made in existing data in zendesk chat.

At Destination:
![image](https://github.com/airbytehq/airbyte/assets/106967803/a7964fd6-f091-41f7-ac09-ad7c58abb261)

At Source:
![image](https://github.com/airbytehq/airbyte/assets/106967803/e6bd3f3c-8874-4192-b415-55e40b54b689)
![image](https://github.com/airbytehq/airbyte/assets/106967803/81e5e30f-c3e9-438f-8912-f6542adf4bce)

Logs:
[zendeskchatlogs.txt](https://github.com/airbytehq/airbyte/files/13887815/zendeskchatlogs.txt)

**Comment by sajarin** [comment]
hey @WisamulHaque, 

this is intended behavior with incremental | append, if you'd like to avoid duplicate data, you will need to use incremental + dedupe if the connector supports it.

[a comment by an automated account is not shown]
